package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "swagger-router",
-  "version": "0.5.6",
+  "version": "0.6.0",
```

Comment:
It does add a new feature though, so imho a minor version bump is justified. Breaking change should be a major version bump, imho.